src/components/playback/mediasession.js

```diff
@@ -54,14 +58,16 @@ import connectionManager from 'connectionManager';
                 src: url,
                 sizes: height + 'x' + height
             };
+        } else {
+            return null;
```

Comment:
won't this break later on when the result of `getImageUrls()` containing `null`s would be used by something else?

Reply:
It would already break since nothing is currently returned.

I added a explicit check to omit pushing the image to the array if it's null.